src/renderer/features/wallets/ImportKeys/ui/ImportKeysModal.tsx

```diff
@@ -92,7 +90,12 @@ export const ImportKeysModal = ({ isOpen, rootAccountId, existingKeys, onConfirm
           ))}
         </Alert>
 
-        <InfoLink url={templateFile} className="gap-x-1 mt-2 px-3" iconName="import" iconPosition="right" download>
+        <InfoLink
+          url="https://github.com/novasamatech/nova-spektr-utils/blob/main/templates/dd-template.yaml"
```

Comment:
I think better to create constant for url

Reply:
Agree, `features/wallets/lib/constants` or just here above the component
And use raw link > `https://raw.githubusercontent.com/novasamatech/nova-spektr-utils/main/templates/dd-template.yaml` otherwise CORS would block it 😉